src/app/login-dialog/login-dialog.component.ts

```diff
@@ -13,25 +13,38 @@
 // limitations under the License.
 
 import {Component, OnInit} from '@angular/core';
+import * as template from './login-dialog.component.html';
+import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+import {MatDialogRef} from '@angular/material';
+import {SessionService} from '../session/session.service';
 import {ActivatedRoute, Params, Router} from '@angular/router';
-import { SessionService } from '../session/session.service';
 
 @Component({
-  selector: 'app-login',
-  templateUrl: './app-login.component.html',
-  styleUrls: ['./app-login.component.scss'],
+  selector: 'app-login-dialog',
+  template: `<div class="container">${template}</div>`,
```

Comment:
Isn't it possible to use `templateURL` and directly reference the template? Would be much better.